content/en/integrations/guide/aws-manual-setup.md

```diff
@@ -0,0 +1,118 @@
+- link: "https://www.datadoghq.com/blog/aws-monitoring/"
+  tag: "Blog"
+  text: "Key metrics for AWS monitoring"
+- link: "https://www.datadoghq.com/blog/monitoring-ec2-instances-with-datadog/"
+  tag: "Blog"
+  text: "How to monitor EC2 instances with Datadog"
+- link: "https://www.datadoghq.com/blog/monitoring-aws-lambda-with-datadog/"
+  tag: "Blog"
+  text: "Monitoring AWS Lambda with Datadog"
+- link: "https://www.datadoghq.com/blog/cloud-security-posture-management/"
+  tag: "Blog"
+  text: "Introducing Datadog Cloud Security Posture Management"
+- link: "https://www.datadoghq.com/blog/datadog-workload-security/"
+  tag: "Blog"
+  text: "Secure your infrastructure in real time with Datadog Cloud Workload Security"
+- link: "https://www.datadoghq.com/blog/announcing-cloud-siem/"
+  tag: "Blog"
+  text: "Announcing Datadog Security Monitoring"
+---
+
+## Overview
+
+Use this guide to manually set up the Datadog [AWS Integration][1].
+
+{{< tabs >}}
+{{% tab "Role delegation" %}}
+
+Setting up the AWS integration manually involves creating an IAM policy and IAM role in your AWS account, and configuring the role with an AWS External ID generated in your Datadog account. This allows Datadog's AWS account to query AWS APIs on your behalf, and pull data into your Datadog account. The sections below detail the steps for creating each of these components, and then completing the setup in your Datadog account.
+
+## Setup
+
+### Generate an external ID
+Generate an External ID in the <a href="https://app.datadoghq.com/account/settings#integrations/amazon_web_services" target="_blank">AWS integration tile</a> on the integrations page. This is used in the trust policy of the AWS IAM role you create for Datadog.
+1. Select the `Configuration` tab, then `Role Delegation`.
+2. Click `Manually`. This creates an AWS External ID which is used for configuration of the AWS IAM role. For more information about the External ID, see the [IAM User Guide][2].
+3. Copy this value to your clipboard or notepad.  
+  **Note: Do not close the integration tile or the Datadog application page**, as this causes the external ID value to reset.
+
+### AWS IAM Policy for Datadog
+Create an IAM policy for the Datadog role in your AWS account with the [necessary permissions](#aws-integration-iam-policy) to take advantage of every AWS integration offered by Datadog. As other components are added to an integration, these permissions may change. 
+
+4. Create a new policy in the AWS [IAM Console][3].
+5. Select the `JSON` tab. Paste the [permission policies](#aws-integration-iam-policy) in the textbox. 
+6. Click `Next: Tags` and `Review policy`.
+7. Name the policy `DatadogAWSIntegrationPolicy` or one of your own choosing, and provide an apt description.
+8. Click `Create policy`.
+
+### AWS IAM Role for Datadog
+Create an IAM role for Datadog to use the permissions defined in the IAM policy.
+
+10. Create a new role in the AWS [IAM Console][5].
```

Comment:
@buraizu step 9 gets skipped.  Goes from 8 -> 10

Reply:
Thanks for pointing this out! Corrected the numbering across all three sections in #66bf0b7.